tests/Report/TextReporterTest.php

```diff
@@ -63,7 +63,10 @@ public function testGenerateNoErrors()
 
     public function testGenerateSimple()
     {
-        $expectedReport = str_replace("\n", PHP_EOL, <<<'TEXT'
+        $expectedReport = str_replace(
+            "\n",
+            PHP_EOL,
+            <<<'TEXT'
```

Comment:
A heredoc / nowdoc cannot end with something else than a newline. This means that any function call containing one should be multiline. This looks right to me.